Paltineanu Rares-Mihai/curs9 - 3.8/main.c

```diff
@@ -0,0 +1,114 @@
+/** Read  from  the standard input the  author,  the  title  and  the  publication  year  for  a  number  of n books.
+Display the following:
+a)the names of the authors in alphabetic order;
+b)the names of the authors and the titles of their books in order of the publication year.
+*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+
+struct probl {
```

Comment:
Thinking ahead, are we!

Reply:
It would be even better to name your structures a bit better as well 😏

Reply:
I dont know why i named the structure like this...